source/nodejs/adaptivecards/src/host-config.ts

```diff
@@ -585,25 +585,25 @@ export class FontStyleDefinition {
     }
 }
 
-export class FontStyleSet {
-    default: FontStyleDefinition;
-    monospace: FontStyleDefinition;
+export class FontTypeSet {
+    default: FontTypeDefinition;
+    monospace: FontTypeDefinition;
 
     constructor(obj?: any) {
-        this.default = new FontStyleDefinition();
-        this.monospace = new FontStyleDefinition("'Courier New', Courier, monospace");
+        this.default = new FontTypeDefinition();
+        this.monospace = new FontTypeDefinition("'Courier New', Courier, monospace");
 
         if (obj) {
```

Comment:
curious: does if(obj) evaluate to if (obj == null) OR if (obj === null) //strict null check?  I am assuming the latter to account for undefined here?

Reply:
AFAIK it is equivalent to `if (obj !== undefined)`

The API isn't compiled with **strictNullCheck** nor **noImplicitAny**. At some point I'll make the changes, but they are non trivial (I have already looked into it.)

Reply:
tx 

---
In reply to: [284927734](https://github.com/microsoft/AdaptiveCards/pull/2905#discussion_r284927734) [](ancestors = 284927734)